pyupgrade.py

```diff
@@ -23,10 +40,29 @@
 from tokenize_rt import UNIMPORTANT_WS
 
 
+Parsed = Tuple[Text, Optional[Text], Optional[Text], Optional[Text]]
+Fmt = Tuple[
```

Comment:
maybe `DotFormatPart` and `PercentFormatPart`?

Reply:
Not following, are you saying to create 2 types? And use them separately from Fmt?

Reply:
oh I meant more that `Parsed` and `Fmt` aren't quite clear what they're being used for

Reply:
Gotcha